dcl-common/src/scene.rs

```diff
@@ -0,0 +1,77 @@
+use serde::{Deserialize, Serialize};
+use super::Parcel;
+
+#[derive(Debug, Deserialize, Serialize, Default, Clone)]
+pub struct Scene {
```

Comment:
We want to add documentation to all the structs we create that represent important data. Scene is one of them. In this case I'm not sure what `path` is.